app/helpers/notices_helper.rb

```diff
@@ -133,4 +133,8 @@ def truncatable_cache_key(key, can_see_full)
       key
     end
   end
+
+  def give_domains_from_urls(infringing_urls_list)
```

Comment:
If this function is used only by `NoticeSubmissionFinalizer`, it should live on that class (and the finalizer in turn doesn't need to include this).

Reply:
Yeah we can have this there, but I though this might be useful somewhere else as well. I will move this to NoticeSubmissionFinalizer!

Reply:
If it becomes useful in multiple places, we can always move it later, and our knowledge of the situation will guide us to make logical choices about where to put it. Right now we don't know where else it might be useful, so it should live in the place we know it's relevant. (This is an idea you'll see referred to as "avoiding premature optimization" or "YAGNI" -- worth a google if you havne't already seen these.)